motoman_driver/src/industrial_robot_client/joint_trajectory_action.cpp

```diff
@@ -159,10 +160,20 @@ JointTrajectoryAction::JointTrajectoryAction() :
 
     this->act_servers_[group_number_int]->start();
 
+    this->act_clients_[group_number_int] = actionClient_;
```

Comment:
This was something we were trying when debugging it is safe for it to be removed.